src/Explorer/Tabs/Tabs.tsx

```diff
@@ -92,6 +92,7 @@ function TabNav({ tab, active, tabKind }: { tab?: Tab; active: boolean; tabKind?
         }
       }}
       className={active ? "active tabList" : "tabList"}
+      style={active ? { fontWeight: "bolder" } : {}}
```

Comment:
Which accessibility bug does this fix? I checked all the bugs listed in the description and this doesn't seem to be related to any of them.

Reply:
This resolves the defect reted to color contrast ratio. Here is the link
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2264174/

Reply:
Thanks Sampath. Even though this is a small fix, please make sure you create a separate PR instead of adding it to this PR which already has more than 5 different fixes.